src/frontends/lean/structure_cmd.cpp

```diff
@@ -654,7 +654,8 @@ struct structure_cmd_fn {
                     expr base_obj = *mk_base_projections(m_env, parent_name, base_S_name, field);
                     level_param_names lparams; unsigned nparams; inductive::intro_rule intro;
                     std::tie(lparams, nparams, intro) = get_parent_info(base_S_name);
-                    expr type = instantiate_univ_params(m_p.env().get(full_fname).get_type(), lparams, const_levels(parent_fn));
+                    levels meta_ls = map2<level>(lparams, [&](name const &) { return m_ctx.mk_univ_metavar_decl(); });
+                    expr type = instantiate_univ_params(m_p.env().get(full_fname).get_type(), lparams, meta_ls);
```

Comment:
@leodemoura I should probably ask whether this whole hack (see also comment above it) is the correct way to go. Basically, all I need is the type `a` to use in a telescope like `Pi (p : applicative), let map : a := functor.map (applicative.to_functor p) in ...`. If I just use a placeholder instead, elaboration fails with some delayed abstractions.